src/panels/KeybindsPanel/KeybindButtons.tsx

```diff
@@ -1,15 +1,19 @@
 import { Fragment } from 'react/jsx-runtime';
-import { Group, Kbd, Text } from '@mantine/core';
+import { Group, Kbd, MantineStyleProps, Text } from '@mantine/core';
 
 import { KeyboardDisplayNames } from './FullKeyboard/data';
 import { keyToOpenSpaceKey } from './FullKeyboard/util';
 
-interface KeybindButtonsProps {
-  modifiers: string[];
-  selectedKey: string;
+interface KeybindButtonsProps extends MantineStyleProps {
+  modifiers?: string[];
+  selectedKey?: string;
 }
 
-export function KeybindButtons({ modifiers, selectedKey }: KeybindButtonsProps) {
+export function KeybindButtons({
+  modifiers = [],
+  selectedKey = '',
+  ...style
```

Comment:
```suggestion
  ...styleProps
```
?  To not confuse it with the `style` argument?  At least, we should try to be consistent across the codebase

There was some other case where I saw this as well, don't remember where now